src/main/java/ca/team2706/frc/robot/config/Config.java

```diff
@@ -93,6 +149,38 @@ public static void init() {
     public static final int TANK_DRIVE_RIGHT = XboxValue.XBOX_RIGHT_STICK_Y.getPort();
     public static final int TANK_DRIVE_LEFT = XboxValue.XBOX_LEFT_STICK_Y.getPort();
 
+    /**
+     * Amount of time (in seconds) that it takes for the plunger to be stowed.
+     */
+    public static final double PLUNGER_TIMEOUT = 0.25;
+    /**
+     * How long the intake motors should be running before the plunger deploys, in seconds.
+     */
+    public static final double EXHALE_CARGO_WAIT_UNTIL_PLUNGER = 0.5;
+
+    /**
+     * How long to wait to ensure the intake arms have fully moved.
+     */
+    public static final double INTAKE_ARMS_DELAY = 1.0;
+
+    /**
+     * How much height (in feet) to subtract from the lift's height for ejecting hatches.
+     */
+    public static final double SUBTRACT_LIFT_HEIGHT = -0.125;
+    /**
+     * How far from the top and the bottom of the lift that the lift should begin to slow down, in manual control.
+     */
+    public static final double LIFT_SLOWDOWN_RANGE_UP = 0.5;
+    public static final double LIFT_SLOWDOWN_RANGE_DOWN = 1.0;
+
+
+    public static final double MAX_INTAKE_SPEED = 1.0; //to be finalized later, this has yet to be tested
+
+    public static final double[] ENCODER_LIFT_PID_UP = {0.5, 0, 50}; //to be finalized later, these values have not yet been tested
```

Comment:
Is this different from the fluid constants below for Lift PID?

Reply:
Actually, neither of them are used at all so I'll just delete them both.